main.cpp

```diff
@@ -31,6 +42,7 @@ namespace TelCoColorCoder
  
         testPairToNumber(TelCoColorCoder::BLACK, TelCoColorCoder::ORANGE, 12);
         testPairToNumber(TelCoColorCoder::VIOLET, TelCoColorCoder::SLATE, 25);
- 
+
+        std::cout<< TelCoColorCoder::ColorCoding();
```

Comment:
Good decision to separate the formation of the color coding manual from its printing.